pkg/services/accesscontrol/database/externalservices.go

```diff
@@ -148,7 +148,15 @@ func getRolePermissions(ctx context.Context, sess *db.Session, id int64) ([]acce
 func permissionDiff(previous, new []accesscontrol.Permission) (added, removed []accesscontrol.Permission) {
 	type key struct{ Action, Scope string }
 	prevMap := map[key]int64{}
+	unSplit := map[key]int64{}
 	for i := range previous {
+		// FIXME: This can be removed after a few releases.
```

Comment:
Technically not 100% needed if we re-introduce the split scope migration.
But that way this PR is self sufficient.
We need to track that in an issue.